Scripts/Level_Components/IDE.gd

```diff
@@ -4,9 +4,12 @@ signal executed
 
```

Comment:
Understandable, but a bit difficult to follow the flow of execution mentally.
Would like an overview/breakdown of this during the weekly meeting.

Possible idea for refactoring put in comments below